src/MockLibraries/Mock.Chemistry/Mock.Chemistry.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.Quantum.Sdk/0.15.2101126099-alpha">
+<Project Sdk="Microsoft.Quantum.Sdk/0.15.2101126099-alpha">
```

Comment:
This is already in by #400, right?

Reply:
Yes, but there was a merge conflict in doing so. Resolving to take #400's changes generated some spurious diffs due to invisible whitespace (namely, the UTF-16 BOM).